app/models/foreman_azure_rm/azure_rm.rb

```diff
@@ -155,7 +180,9 @@ def vm_sizes(region)
     end
 
     def vm_instance_defaults
-      ActiveSupport::HashWithIndifferentAccess.new
+      super.merge(
```

Comment:
`deep_merge` suits better - if there are some properties defined on interfaces in `super`, we still want them. Regular `merge` will replace the whole `interfaces` member.

Reply:
done.